### DISCUSSION TITLE
Allowed to have empty billing plans list?

### DISCUSSION THREAD

**Discussion by soxofaan** [Open-EO/openeo-api#420] [General]
https://github.com/Open-EO/openeo-api/blob/592e2662c23843745e2e8c28269b19685cf13cd1/openapi.yaml#L746-L750

just checking: is it intentional that the billing plans array can be empty?

**Comment by soxofaan** [comment]
(first time I use this "discussion" feature on github which is probably better for these kind of questions compared to  creating an issue)

**Reply by m-mohr** [reply]
I have nothing against using issues for such things, to be honest.

**Reply by soxofaan** [reply]
This threading aspect  could be handy however for larger discussions (compared to the single thread  nature of issues)

**Reply by m-mohr** [reply]
Sure, although many (of the smaller) issues (like this one) you usually report lead to issues and PRs anyway (which is meant in a positive way). ;-)

**Comment by m-mohr** [comment]
Hmm, I'm not sure but right now I don't really see a reason why we couldn't set "minItems" to 1 except that it would be somewhat breaking.

So yes right now it can be empty, but it is not very useful and basically says: No billing available at the back-end.

**Reply by m-mohr** [reply]
See also https://github.com/Open-EO/openeo-api/issues/422

**Reply by soxofaan** [reply]
the `plans` array can also be omitted as a whole, which means the same thing I guess

**Reply by m-mohr** [reply]
Yes, indeed.
